### ISSUE TITLE
Android adaptive-icon blurry on some devices

### ISSUE THREAD

**Issue by ontech7** [expo/expo#19881]
### Summary

When using adaptive-icon for Android, I'm facing this issue since the beginning of development.

The icon is blurry.

The device I'm using has this software specs:

Poco F2 Pro (Xiaomi) - MIUI 13.0.5 Global - Android 12 SKQ1.211006.001

### What platform(s) does this occur on?

Android

### Environment

```
System:
      OS: Windows 10 10.0.19043
    Binaries:
      Node: 16.17.1 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
      npm: 8.15.0 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: AI-213.7172.25.2113.9123335
    npmPackages:
      expo: ~46.0.16 => 46.0.16
      react: 18.0.0 => 18.0.0
      react-native: 0.69.6 => 0.69.6
    Expo Workflow: managed
```

### Minimal reproducible example

I created an example here with a test icon: https://github.com/ontech7/expo-blurry-adaptive-icon

I already put all the icons using Figma Guidelines, that I can share it as well, if you want to recreate the issue by yourself: https://www.figma.com/file/wzNGjy9AYA9chPkDG8wMeY/Untitled

I'm using this profile to build (I already created the `android:build` command runnable using `npm run android:build`)

```
"preview": {
      "android": {
        "buildType": "apk"
      }
    },
```

The app.json is inside here: https://github.com/ontech7/expo-blurry-adaptive-icon/blob/main/app.json

And dependences are:

```
"expo": "~46.0.16",
"react": "18.0.0",
"react-native": "0.69.6"
```

![Screenshot_2022_11_05_22_44_32_150_com_mi_android_globallauncher](https://user-images.githubusercontent.com/3172757/200142464-0612ed9e-8d8b-401a-b16c-766d8e89ce97.jpg)

**Comment by hanzketup** [comment]
We're facing the same issue and i've found that the problem is easily reproducible on a simulated Android 13 device.

Steps to reproduce:

1. Clone [this repository](https://gitlab.com/hanzketup/expo-blurry-adaptive-icon-reprod/-/tree/main)
2. Spin up an emulated Pixel 6 with Android 13 (Tiramisu) from Android studio
3. run `npx expo run:android`
4. Observe the issue: blurred icon on the home screen and app drawer, and a very blurry icon when the app is being opened.

**Comment by hanzketup** [comment]
@brentvatne you mentioned your frustration over the lack of reproducible examples of this issue on Discord, hopefully these two help 🙌

**Comment by Julioevm** [comment]
I have the same issue. I get the same results building an apk or a full aab bundle for release, if that's any use.

**Comment by ontech7** [comment]
Hi @Julioevm can you tell us your specs (Android phone name, Android version, etc.)?

**Comment by Julioevm** [comment]
Sure, I'm using a Xiaomi Redmi 10 Pro (MIUI 13,0,15, Android 12) also tried on a Redmin 7 with Android 10 and on an emulator also with Android 10.

**Comment by brentvatne** [comment]
https://github.com/expo/expo/pull/19948#issuecomment-1326919044

> [a fix for this] was released in expo@47.0.4. i recommend updating to the latest expo@47 version, which at the time of writing is 47.0.8. update the version in your package.json then run npm install or yarn again
